README.md

```diff
@@ -44,9 +44,11 @@ If you need to add features to the API, the best is to [`npm link`](https://docs
 Openfisca
 ---------
 
+Openfisca relies on python 2. You should [install it in a `virtualenv`](https://doc.openfisca.fr/for_developers.html). Once activated, you can run the following commands:
```

Comment:
The referenced links is not very precise. If possible, make a direct link to the virtual reference. If not and you're ready to update this README whenever the linked documentation changes, then at least link to the [proper paragraph](https://doc.openfisca.fr/for_developers.html#create-a-virtualenv) :wink:

I strongly recommend that the activation of the virtualenv is listed in the commands list below rather than as a text-only “once activated” instruction.